cmd/main.go

```diff
@@ -104,37 +106,36 @@ func main() {
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "80807133.rcs.dana.io",
 	})
-
 	if err != nil {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
 
-	if err = (&controllers.CappPlacementReconciler{
+	if err = (&syncctrl.SyncReconciler{
 		Client:        mgr.GetClient(),
 		Scheme:        mgr.GetScheme(),
-		EventRecorder: mgr.GetEventRecorderFor("cappPlacementSync_controller"),
+		EventRecorder: mgr.GetEventRecorderFor("sync_controller"),
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "Capp")
 		os.Exit(1)
 	}
 
-	if err = (&controllers.CappNamespaceReconciler{
+	if err = (&placementctrl.PlacementReconciler{
 		Client:        mgr.GetClient(),
 		Scheme:        mgr.GetScheme(),
-		EventRecorder: mgr.GetEventRecorderFor("cappPlacement_controller"),
+		EventRecorder: mgr.GetEventRecorderFor("placement_controller"),
```

Comment:
Not sure if there's a rule here but Kubebuilder uses `-` as opposed to `_` in their example https://book.kubebuilder.io/reference/raising-events